.github/workflows/swift.yml

```diff
@@ -9,15 +9,14 @@ on:
 jobs:
   test:
 
-    runs-on: macos-12
+    runs-on: macos-13
 
     steps:
     - uses: actions/checkout@v3
     - name: Install swift-format
       run: |
         brew update
         brew install swift-format
-        brew upgrade
```

Comment:
It looks like the latest package upgrade state for Homebrew has internally inconsistent dependencies. It's likely Homebrew upgrades will fail again, so it's safest to just use the CI defaults. Although packages will be slightly out of date, it shouldn't be a big deal.

Reply:
🙏 🙏 SGTM ser.